VideoWeb/VideoWeb.Contract/Responses/ConferenceResponse.cs

```diff
@@ -71,5 +71,10 @@ public class ConferenceResponse
         /// The participants in the conference
         /// </summary>
         public List<ParticipantResponse> Participants { get; set; }
+
+        /// <summary>
+        /// Closed date time as UTC
+        /// </summary>
+        public DateTime ClosedDateTime { get; set; }
```

Comment:
Keep this as nullable since it isn't guaranteed to have a value and defaults to 1/1/1